tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -12,12 +12,12 @@
 from rest_framework.viewsets import ModelViewSet
 from rest_framework.decorators import action
 from wsgiref.util import FileWrapper
+from rest_framework import status
 
 from tdpservice.data_files.serializers import DataFileSerializer
 from tdpservice.data_files.models import DataFile
 from tdpservice.users.permissions import DataFilePermissions
-
-logger = logging.getLogger()
+from tdpservice.scheduling import tasks
```

Comment:
As we discussed, we will rename `tasks.py` to a descriptive `sftp_tasks.py`.

Reply:
For posterity, we decided on an architecture for forward-looking celery tasking that we won't have a monolithic tasks file with all celery tasks but will organize them into their own files as in option 2 below.

```
1.
tdpservice/
	data_files/
		views.py
		celery_tasks.py
	users/
		celery_tasks.py
	elasticsearch/
		celery_tasks.py


vs
2.
tdpservice/
	data_files/
		views.py
	scheduling/
		data_files_tasks.py
		user_tasks.py
		elasticsearch_tasks.py

vs

3.
tdpservice/
	data_files/
		views.py
	scheduling/
		tasks.py
```